certsuite/report/report.css

```diff
@@ -20,6 +20,7 @@ body {
 }
 .PASS { background-color: green }
 .FAIL { background-color: red }
+.OK { background-color: lightgreen}
```

Comment:
I'll suggest to use "EXECUTED" for bringing correct images. "OK" sounds like it get passed.